ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -398,6 +401,33 @@ - (NSDictionary *)powerState
     resolve(@(locationServicesEnabled));
 }
 
+RCT_EXPORT_METHOD(getUserAgent:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
This looks the same as the current https://github.com/react-native-community/react-native-device-info/blob/6f76bde4e84f4ccda2fda4e1aa26c9dc0e63ed73/ios/RNDeviceInfo/RNDeviceInfo.m#L707